src/NuGet.Core/NuGet.Protocol/Plugins/Logging/PluginLogMessage.cs

```diff
@@ -32,7 +33,7 @@ protected string ToString(string type, JObject message)
             }
 
             var outerMessage = new JObject(
-                new JProperty("now", _now.ToString("O")), // round-trip format
+                new JProperty("now", _now.ToString("O", CultureInfo.CurrentCulture)), // round-trip format
```

Comment:
Round trips should use the same format:

```suggestion
                new JProperty("now", _now.ToString("O", CultureInfo.InvariantCulture)), // round-trip format
```